README.adoc

```diff
@@ -302,6 +304,53 @@ If you are using VisualStudio Code, you can install the following `launcher.json
 }
 ----
 
+==== GUI Frontend
+
+If you are using VisualStudio Code, you can install the following `launcher.json` that is then used to launch Chrome to debug the Kiali Server GUI frontend in the debugger. Run the `make -e YARN_START_URL=http://localhost:20001/kiali yarn-start` first to start the GUI frontend locally. Make sure your YARN_START_URL is valid - it should point to a running Kiali Server. In this example, it assumes you ran a port-forward so that the Kiali Server can be reached at localhost:20001. If running in minikube, you could alternatively use `$(minikube ip):80` as the host/port assuming an Ingress is set up to access Kiali.
+
+[source,json]
+----
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Launch Chrome",
+            "type": "chrome",
+            "request": "launch",
+            "url": "http://localhost:3001",
```

Comment:
Here I will change it for 3000, given that is the original port, the 3001 I think it's for the way I'm creating tunnels.

Reply:
If we tell people to use 3000, won't that break if they are running the backend via "run-kiali.sh" (which port-forwards grafana on 3000???)

If we use 3001, I think that would be safe no matter how you run the backend, right?

Reply:
I added some more around this whole 3000 vs 3001.